src/classes/User/User.spec.ts

```diff
@@ -43,24 +40,33 @@ describe('User class', () => {
     });
 
     it('should throw an error if get user by identifier fails', async () => {
-        usersApiMock.listPaginatedUsers.mockRejectedValue(
+        const responseError = new ResponseError(
+            {
+                status: 502,
+                json: async () => ({ code: 'error_code', error: 'Bad gateway' }),
+            } as Response,
+            'Error',
+        );
+        const passageError = await PassageError.fromResponseError(responseError);
```

Comment:
setting this up just to keep the assert on line 53 that the thrown error is an instance of `PassageError`

using `rejects.toThrow(PassageError)` doesn't work because `PassageError` is no longer "constructable" due to the private constructor